src/rules/indentation.js

```diff
@@ -33,30 +33,39 @@ function testStep(step, language, configuration, mergedConfiguration) {
 }
 
 function indentation(parsedFile, unused, configuration) {
+  if (!parsedFile || Object.keys(parsedFile).length === 0) {
+    return;
+  }
   var language = languageMapping[parsedFile.language];
   var mergedConfiguration = _.merge(availableConfigs, configuration);
   errors = [];
 
   // Check Feature indentation
   test(parsedFile.location, mergedConfiguration, 'Feature');
 
-  if (parsedFile.background) {
-    // Check Background indentation
-    test(parsedFile.background.location, mergedConfiguration, 'Background');
-
-    // Check Background steps
-    parsedFile.background.steps.forEach(function(step) {
-      testStep(step, language, configuration, mergedConfiguration);
-    });
-  }
+  parsedFile.children.forEach(function(child) {
+    switch(child.type) {
+    case 'Background':
+      // Check Background indentation
+      test(child.location, mergedConfiguration, 'Background');
 
-  parsedFile.scenarioDefinitions.forEach(function(scenario) {
-    // Check Scenario indentation
-    test(scenario.location, mergedConfiguration, 'Scenario');
-    scenario.steps.forEach(function(step) {
-      // Check Step indentation
-      testStep(step, language, configuration, mergedConfiguration);
-    });
+      // Check Background steps
+      child.steps.forEach(function(step) {
```

Comment:
You can pull this code outside the switch, it's the same for Background and Scenario now.